test/Intl/NumberFormatOptions.js

```diff
@@ -82,18 +82,20 @@ var tests = [
         name: "Test Valid Options - Formatting Currency with Significant Digits",
         body: function () {
             const usdBaseOptions = { minimumSignificantDigits: 3, maximumSignificantDigits: 3, style: "currency", currency: "USD" };
-            var expectedRegex1 = new RegExp("\\$[\x20\u00a0]?123");
-            var expectedRegex2 = new RegExp("USD[\x20\u00a0]?123");
+            const symbolFormat = /\$[\x20\u00a0]?123/;
```

Comment:
Do you need to update tags in `test/Intl/rlexe.xml`?

Reply:
I thought I already did, but this comment may have been from before I committed that. I updated a bunch of tags, though this test was already not marked as require_winglob, so it didn't need to be updated.